cymetric/metrics.py

```diff
@@ -280,6 +280,10 @@ def transaction_quantity(series):
 ## FCO-related metrics ##
 #########################
 
+# Some useful variables
+x_feed = 0.0072
+x_tails = 0.0025
```

Comment:
Or if you want them here they should be defined as pep8 compliant constants. Eg `X_HEAD`